behave/model.py

```diff
@@ -1767,6 +1767,19 @@ def run(self, runner, quiet=False, capture=True):
         # self.hook_failed = False
         self.reset()
 
+        flat_context = runner.context.as_flat()
+
+        for k in list(flat_context.keys()):
+            try:
+                flat_context[k] = str(k)
+            except:
```

Comment:
Using except without Exception is BAD.

Reply:
true! I'll fix this once we agree on a goal and syntax.